### ISSUE TITLE
nested routes

### ISSUE THREAD

**Issue by pablodv** [museways/translatable_routes#4]
I have problems with nested routes:

```ruby
    resources :lotteries, only: [:index, :show] do
      collection{ get 'results' }
      member{ get 'expired', to: 'lotteries#expired' }

      resources :draws, path: 'results', only: [:index] do
        get "details", to: "draws#show"
      end
    end
```

return:

```ruby
      lottery_fi_draw_details GET      /:locale/lotteries/:lottery_id/results/:draw_id/details(.:format)    draws#show {:locale=>"fi"}
                                GET      /:locale/lotteries/:lottery_id/results/:draw_id/details(.:format)    draws#show {:locale=>"de"}
       lottery_fi_draw_detalles GET      /:locale/loterias/:lottery_id/resultados/:draw_id/detalles(.:format) draws#show {:locale=>"es"}
        lottery_fi_draw_detalii GET      /:locale/loterii/:lottery_id/rezultate/:draw_id/detalii(.:format)    draws#show {:locale=>"ro"}
       lottery_fi_draw_dettagli GET      /:locale/lotterie/:lottery_id/risultati/:draw_id/dettagli(.:format)  draws#show {:locale=>"it"}
                                GET      /:locale/lotteries/:lottery_id/results/:draw_id/details(.:format)    draws#show {:locale=>"tr"}
         lottery_fi_draw_detali GET      /:locale/loterei/:lottery_id/rezultaty/:draw_id/detali(.:format)     draws#show {:locale=>"ru"}
                                GET      /:locale/loterijen/:lottery_id/resultaten/:draw_id/details(.:format) draws#show {:locale=>"nl"}
        lottery_fi_draw_detaily GET      /:locale/loterie/:lottery_id/vysledky/:draw_id/detaily(.:format)     draws#show {:locale=>"sk"}
                                GET      /:locale/lotteries/:lottery_id/results/:draw_id/details(.:format)    draws#show {:locale=>"en"}
            lottery_fi_draws_fi GET      /:locale/lotteries/:lottery_id/results(.:format)                     draws#index {:locale=>"fi"}
            lottery_fi_draws_de GET      /:locale/lotteries/:lottery_id/results(.:format)                     draws#index {:locale=>"de"}
            lottery_fi_draws_es GET      /:locale/loterias/:lottery_id/resultados(.:format)                   draws#index {:locale=>"es"}
            lottery_fi_draws_ro GET      /:locale/loterii/:lottery_id/rezultate(.:format)                     draws#index {:locale=>"ro"}
            lottery_fi_draws_it GET      /:locale/lotterie/:lottery_id/risultati(.:format)                    draws#index {:locale=>"it"}
            lottery_fi_draws_tr GET      /:locale/lotteries/:lottery_id/results(.:format)                     draws#index {:locale=>"tr"}
            lottery_fi_draws_ru GET      /:locale/loterei/:lottery_id/rezultaty(.:format)                     draws#index {:locale=>"ru"}
            lottery_fi_draws_nl GET      /:locale/loterijen/:lottery_id/resultaten(.:format)                  draws#index {:locale=>"nl"}
            lottery_fi_draws_sk GET      /:locale/loterie/:lottery_id/vysledky(.:format)                      draws#index {:locale=>"sk"}
            lottery_fi_draws_en GET      /:locale/lotteries/:lottery_id/results(.:format)                     draws#index {:locale=>"en"}
```

translate the helper and put the first locale in the middle.

**Comment by mmontossi** [comment]
I'll be hacking at this soon, thanks for sharing the bug!

**Comment by voondo** [comment]
sorry but it produces named routes like "foo_en_bar_fr"

**Comment by mmontossi** [comment]
@voondo can you post the routes you're using? so I can test against those.

**Comment by voondo** [comment]
```
resources :albums, only: %i(index destroy) do
  resources :photos, only: %i(destroy), controller: 'albums/photos'
end
```
produces : 
```
      album_en_photo_en DELETE /:locale/albums/:album_id/photos/:id(.:format)      albums/photos#destroy {:locale=>"en"}
      album_en_photo_fr DELETE /:locale/albums/:album_id/photos/:id(.:format)      albums/photos#destroy {:locale=>"fr"}
```

**Comment by mmontossi** [comment]
@voondo and @pablodv I just released a new version 1.4.0, it fixes this issue once and for all.

Thanks for sharing
